app/src/main/java/com/example/footprnt/Discover/Adapters/ListAdapter.java

```diff
@@ -70,7 +72,8 @@ public int getItemCount() {
     }
 
     public static class BusinessViewHolder extends RecyclerView.ViewHolder {
-        public View cardView;
+        // TODO: change names to fit android style
+        View cardView;
```

Comment:
Inner classes are in a gray area but I'd still like to see you following the naming convention of mVariableName. Also often inner classes are unnecessary and just making this its own class is advised